app/controllers/katello/api/v2/activation_keys_controller.rb

```diff
@@ -99,7 +99,8 @@ def copy
       @new_activation_key.reload
       @new_activation_key.update_attributes!(
                                              :service_level => @activation_key.service_level,
-                                             :release_version => @activation_key.release_version
+                                             :release_version => @activation_key.release_version,
+                                             :auto_attach => @activation_key.auto_attach
```

Comment:
ah, i see why, we should probably dynflow this entire process at some point, can you file an issue for that?